### DISCUSSION TITLE
Languages dynamically

### DISCUSSION THREAD

**Discussion by ya0** [monkeytypegame/monkeytype#2296] [Ideas]
Hi, I love monkeytype. The one feature I always think about when using it, is to get words of a language by their frequency. i.e. Instead of "English 1k" and "English 5K" we could have a category "English general" which simulates words by their frequency in texts.

I have no idea how these modes are implemented. But maybe an easy extension is to use "English" for 70% words, "English 1k" for 70% of the remaining 30% then "English 5k" for 70% of 9% (30% of 30%) of words and the rest take from "English 25k". I hope this description makes some sense, the percentage numbers I have chose are completely arbitrary. Some exponential decaying distribution makes sense!

Maybe you have more insight over the frequency in which words appear in English, which could also be useful to create such a mode.

**Comment by frogfrogfrogfrogfrog** [comment]
Aren't the word categories chosen by how frequent the word appears in the English language? Hence English being the top 100 words while English 1k would be the top 1000 English words in the English language.
